apps/mobile/src/features/send/send-form/hooks/use-send-form-btc.tsx

```diff
@@ -53,11 +51,12 @@ function parseSendFormValues({
     ],
   };
 }
-interface GetTxFeesArgs {
-  feeRates: AverageBitcoinFeeRates;
-  recipients: CoinSelectionRecipient[];
-  utxos: CoinSelectionUtxo[];
-}
+// TODO LEA-2028 - revise fees - this was here but unused
```

Comment:
I'm commenting out this fee stuff as it wasn't doing anything yet but I guess it's needed for something soon, maybe for LEA-2028? 

`getBitcoinFees` is used in the flow for `psbt-signer` which is the next step after this form so it could be just not needed here at all

Reply:
It may have been something I did early to get things going, but maybe fees were moved to the approver flow once Edgar took over?

Reply:
OK, thanks!

I'm just going to delete it to clean up some noise and we can bring it back if we need to